### DISCUSSION TITLE
Problems With PPLNS (Please HELP!)

### DISCUSSION THREAD

**Discussion by hjonathan2019** [coinfoundry/miningcore#1484] [General]
Dear all, good afternoon.
I have a problem with the payment scheme of my pool. I am currently using PPLNS, the problem I am getting is the following:

Let's suppose that a small miner (600 mhs in ergo), took 10 days to collect 1000 shares, and then a large miner (3/4/5 ghs) connects and manages to collect 1000 shares in one day, and then leaves the pool.

A block appears, everyone happy, but that big miner who left the pool, still charges the same portion of it, without being in the pool.

The way I see it, I think this big miner is "stealing" the small miner who has been connected to the pool for a long time and with little mining power supporting the project.

Is there a way that if the miner disconnected from the pool and left, that value of N shares contributed are redistributed to the pool and that they are not paid to the miner who disconnected???

Basically I would like to delete the submitted shares which are still valid for some time after a miner disconnects or maybe redistribute them

**Comment by MiningCryptoLive** [comment]
Unfortunately that is how pplns works.

**Reply by hjonathan2019** [reply]
ok, does miningcore support PPS+ ??

**Comment by MiningCryptoLive** [comment]
Not to my knowledge
